README.md

```
@@ -2,313 +2,1419 @@
+- [Loc Le](https://looc.dev)
+- [Long Do](https://longpdo.github.io)
+- [Loránd Kiss](https://lorandkissdev.github.io/github-portfolio)
+- [Luca Lischetti](https://sirlisko.com)
+- [Lucas Batista](https://lucasbatista.online)
+- [Lucas Dantas](https://dantsdev.vercel.app)
+- [Lucas De Moraes](https://devlucas.website) [Full Stack Developer]
+- [Lucas Lima](https://lucas-lima.vercel.app)
+- [Luis Cacho](https://luiscachog.io)
+- [Luiz Gustavo Alves](https://www.luizgustavoalves.dev) [Full Stack Developer]
+- [Lukasz Kups](https://lukaszkups.net)
+- [Luís Oliveira](https://lo-devpage.netlify.app)
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://pulik.dev)